install

```diff
@@ -137,26 +142,32 @@ do_install()
   pushd src
     # Avoid downloading opencv3; we already installed it from homebrew.
     wstool merge file://$(pwd)/../${WS}.rosinstall
-    wstool remove opencv3
+    #wstool remove opencv3
     wstool update -j8
 
-    # See: https://github.com/ros/catkin/pull/784
-    if [ -d catkin ]; then
-      curl https://raw.githubusercontent.com/ros/catkin/8a47f4eceb4954beb4a5b38b50793d0bbe2c96cf/cmake/catkinConfig.cmake.in > catkin/cmake/catkinConfig.cmake.in
+    if [ -d bond_core ]; then
+      curl curl https://raw.githubusercontent.com/ros/bond_core/1240bce50ca1538cdad07dc472d31797257e0851/bondcpp/src/bond.cpp > bond_core/bondcpp/src/bond.cpp
     fi
```

Comment:
This double `curl` doesn't look quite right. Please fix that and put in a comment indicating what we're waiting on for the patch to go away (in this case, bond_core 1.8.3 to be released, since https://github.com/ros/bond_core/pull/37 is already merged).

Reply:
It's weird it works like that. Will fix it and add comments indicating the releases.